operator/pkg/apis/operator/v1alpha1/type.go

```diff
@@ -793,6 +799,22 @@ type LocalSecretReference struct {
 	Name string `json:"name,omitempty"`
 }
 
+// PodDisruptionBudgetConfig defines a subset of PodDisruptionBudgetSpec fields
+// that users can configure for their control plane components.
+type PodDisruptionBudgetConfig struct {
```

Comment:
Great! Can you also add this new field to our [sample CR](https://github.com/karmada-io/karmada/blob/release-1.15/operator/config/samples/karmada-sample.yaml#L3)?